sno/conflicts.py

```diff
@@ -12,6 +14,113 @@
 L = logging.getLogger("sno.conflicts")
 
 
+class ConflictIndex:
```

Comment:
Is it worth implementing `__eq__` so you can compare two ConflictIndexes? It'd be sane to do that in the roundtrip test you've got. Presumably it'd just compare `entries` and `conflicts`

Reply:
In the end, I just stopped using IndexEntrys internally - in favour of a namedtuple with exactly the same content. Then __eq__ and __repr__ got a lot easier - otherwise they were quite a mess.